README.md

```diff
@@ -1,66 +1,55 @@
 # Cap'n Proto bindings for Go
 
-[![GoDoc](https://godoc.org/zombiezen.com/go/capnproto2?status.svg)][godoc]
-[![Build Status](https://travis-ci.org/capnproto/go-capnproto2.svg?branch=master)][travis]
+[![GoDoc](https://godoc.org/capnproto.org/go/capnp/v3?status.svg)][godoc]
+![License](https://img.shields.io/badge/license-MIT-brightgreen?style=flat-square)
+![tests](https://github.com/capnproto/go-capnproto2/workflows/Go/badge.svg)
 
-go-capnproto consists of:
-- a Go code generator for [Cap'n Proto](https://capnproto.org/)
-- a Go package that provides runtime support
-- a Go package that implements Level 1 of the RPC protocol
+[Cap’n Proto](https://capnproto.org/) is an insanely fast data interchange format similar to [Protocol Buffers](https://github.com/protocolbuffers/protobuf), but much faster.
 
-[godoc]: https://godoc.org/zombiezen.com/go/capnproto2
-[travis]: https://travis-ci.org/capnproto/go-capnproto2
+It also includes a sophisticated RPC system based on [Object Capabilities](https://en.wikipedia.org/wiki/Object-capability_model), ideal for secure, low-latency applications.
 
-## Getting started
+### This package provides:
```

Comment:
Nitpick: jumping from an `<h1>` to an `<h3>` is a bit ugly. I don't think this necessarily needs to be a heading, even.